test/elm/clinical/test.coffee

```diff
@@ -46,6 +46,12 @@ describe 'InValueSet', ->
   it 'should find string code in value set', ->
     @string.exec(@ctx).should.be.true()
 
+  it 'should throw an error when codes are in several codesystems', ->
+    should(() => @sharedCodesFoo.exec(@ctx)).throw('In (valueset) is ambiguous -- multiple matches for foo found in value set with different code systems.')
+
+  it 'should find string code in valueset and not error', ->
+    @properSharedCodesCodeValue.exec(@ctx).should.be.true()
```

Comment:
According to spec this should also throw since this value set covers more than one code system (see my comment on the implementation).